index.html

```diff
@@ -1,63 +1,154 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Sajjda Portfolio</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
 
-  <body>
-    <header id="main-header">
-      <div class="toolbarContainer">
-        <h2 id="logo">Sajjad</h2>
-        <nav>
-          <img src="images/menu-icon.png" id="menu-icon" alt="menu icon" />
-        </nav>
-      </div>
-    </header>
-    <section id="mainSection">
-      <article>
-        <section class="Introduction">
-          <div class="IntroContainer">
-            <div id="primary-text">I’m Sajjad Glad to see you!</div>
-            <div id="secondary-text">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Sajjad Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header>
+    <a href="#">
+      <h2 id="logo">Sajjad</h2>
+    </a>
+    <nav id="mobile-nav">
+      <img src="images/menu-icon.png" id="menu-icon" alt="menu icon" />
+    </nav>
+    <nav id="desktop-nav">
+      <ul>
+        <li>Portfolio</li>
```

Comment:
- [ ] Nice working on your header, kindly link the menu using `a` tag just like the previous reviewer requested.